interface/login/login.php

```diff
@@ -291,8 +299,21 @@ function transmit_form(element) {
                 </div>
             <?php } // End facilities menu block ?>
             <div class="form-group oe-pull-away">
-                <button type="submit" class="btn btn-login btn-lg" onClick="transmit_form(this)"><i class="fa fa-sign-in-alt"></i>&nbsp;&nbsp;<?php echo xlt('Login');?></button>
+                <button id="login-button" type="submit" class="btn btn-login btn-lg" onClick="transmit_form(this)"><i class="fa fa-sign-in-alt"></i>&nbsp;&nbsp;<?php echo xlt('Login');?></button>
             </div>
+            <?php if ($GLOBALS['google_signin_enabled']) { ?>
```

Comment:
if google-signin-client_id needs to be unempty for this to work then would also ensure this in above conditional

Reply:
fixed